### DISCUSSION TITLE
i did wormhole transaction from polygon to bsc

### DISCUSSION THREAD

**Discussion by killer-shot** [wormhole-foundation/wormhole#1268] [Problems / bugs] [answered]
i did a transaction from polygon to bsc usdc
polygon tx confirmed but im unable to redeem from bsc
https://polygonscan.com/tx/0xf57f9c956c78f2b4d84f50b6c50c93c73d416da70136934e2ee2683f2684a817

**Comment by evan-gray** [comment, marked as answer]
Since this post, this transaction has been redeemed.
